[Supported Rails < 4.2.0 ]

ElamT opened the pull request:
- Corrected response_options mergin type opts instead options
  - Responder gem without specific version, so it's compatible with rails version less to 4.2
  - Tested on rails 4.0.0, 4.0.2 and 4.2.0

ElamT committed:
IMPORTANT set config language to Report

ElamT committed:
Update to use Rails 4.2 and update lib jasperreporst to version 5.6.0 and groovy-all-2.0.1.jar

IMPORTANT set Language = Java into reports configuration

ElamT committed:
[Supported Rails < 4.2.0 ]

- Corrected response_options mergin type opts instead options
- responder gem without specific version, so it's compatible with rails version less to 4.2
- Tested on rails 4.0.0, 4.0.2 and 4.2.0

rodrigomaia commented:
Hi, @ElamT 

I found only one error on my tests: undefined local variable or method `opts' for #<ActionController::Responder:0x007ff880935438>

I believe there's no 'opts' anywhere in the code:

-  response_options = JasperRails.config[:response_options].merge(:type => options[:mime_type])
+ response_options = JasperRails.config[:response_options].merge(:type => opts[:mime_type])

I changed to "options" and worked, is there any reason why you changed it?

ElamT commented:
Hi @rodrigomaia opts is one param of the method self.define_render_rails_less_4_2. it is strange not work. I only named it opts there any particular reason.

ElamT committed:
[Support Rails Version 4.x]

- Improvement to test
- Refactor code for support any version of Rails 4.x
- Supporting Chart in Reports

ElamT committed:
Using Rspec Helper gem updated without deprecations

ElamT committed:
[RakeFile default task]

- Add default task rspec (TestSuite)

ElamT committed:
[Travis Conf]

- Serverals enviroments
- Install fonts necesary to generate reports

ElamT committed:
Update README.md

ElamT committed:
[Travis Install Arial Font]

- Install ttf-mscorefonts package

ElamT committed:
Merge branch 'master' of github.com:ElamT/jasper-rails

ElamT committed:
[Travis add Repository Ubuntu]

- add repository multiverse to install fonts Arial ttf-mscorefonts

ElamT committed:
[Change Font Report]

- New font Family

p404 committed:
Update .travis.yml

p404 committed:
Update index.jrxml

p404 committed:
Update travis.yml reqs

ElamT committed:
Merge pull request #1 from p404/master

Fix travis.yml build

ElamT committed:
[Helvetica to Arial]

ElamT committed:
[Ignore index.jasper]

zerobatu committed:
Added expliclit use of jasper in the render
- add in the defult_rederer class a new render (jasperpdf) this do the
  same of pdf but allow the use of multiple pdf gems like wicked_pdf

ElamT committed:
Merge pull request #2 from zerobatu/master

Added expliclit use of jasper in the render
